rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -32,8 +32,8 @@ paths:
     parameters:
       - $ref: '#/components/parameters/AcceptVersion'
     post:
-      summary: Transformation request
-      description: A request to transform a file into discovery metadata and other supporting products.
+      summary: Submit transformation request
+      description: A request to transform an intel resource into discovery metadata and other supporting products.
       operationId: transform
       tags:
         - transform
```

Comment:
❗️ Based on the change pf this API, I am thinking we should not be returning 202 here but instead a 201 for CREATED since we have created a resource for the request which will be retrievable via the other endpoint.